sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/BuilderHelper.java

```diff
@@ -43,14 +43,15 @@
      *
      * @param connectionString Connection string from the service account.
      * @param accountNameSetter Callback to set the account name on the builder.
-     * @param credentialSetter Callback to set the {@link SharedKeyCredential} of the builder.
+     * @param credentialSetter Callback to set the {@link StorageSharedKeyCredential} of the builder.
      * @param endpointSetter Callback to set the endpoint of the builder.
      * @param logger {@link ClientLogger} used to log any exceptions.
      * @throws NullPointerException If {@code connectionString} is {@code null}.
      * @throws IllegalArgumentException If {@code connectionString} doesn't contain 'AccountName' or 'AccountKey'.
      */
     static void configureConnectionString(String connectionString, Consumer<String> accountNameSetter,
-        Consumer<SharedKeyCredential> credentialSetter, Consumer<String> endpointSetter, ClientLogger logger) {
+                                          Consumer<StorageSharedKeyCredential> credentialSetter,
```

Comment:
This was done using the indentation because all the parameters were pressed against the right margin. I'd revert to using an 8 space indent.